index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/4e96f49cc5.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <a href="#"><h1 class="nav-branding">My Logo</h1></a>
+        <ul class="nav-menu">
+          <li class="nav-item nav-link"><a href="#">Portfolio</a></li>
+          <li class="nav-item nav-link"><a href="#">About</a></li>
+          <li class="nav-item nav-link"><a href="#">Contact</a></li>
+        </ul>
+        <div class="hambuger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <!-- Headline -->
+
+    <section class="headline-bg headline">
+      <h1 class="headline-main">
+        I'm Promise <br />
+        Glad to see you!
+      </h1>
+      <p class="headline-paragraph">
+        I’m a software developer! I can help you build a product , feature or
+        website. Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3 class="headline-connect">LET'S CONNECT</h3>
+      <ul class="headline-icons">
+        <li class="fa-brands fa-twitter fa-xl"></li>
+        <li class="fa-brands fa-linkedin-in fa-xl"></li>
+        <li class="fa-brands fa-github fa-xl"></li>
+        <li class="fa-solid fa-m fa-xl"></li>
+        <li class="fa-solid fa-hand-peace fa-xl"></li>
```

Comment:
- [ ] [_Optional_] You might want to make this an `a` tag instead since these are actually external social links. Additionally, the social website for the M is called Medium and the Peace sign is actually AngelList. Both are sites you would be using at Microverse and have their icons available by those names in Font Awesome. Kindly replace them with the specified icons.

Reply:
@zhadier All Changes have been effected.